.github/workflows/dockerimages.yml

```diff
@@ -0,0 +1,40 @@
+name: Docker Images CI
+
+on: [pull_request]
+
+env:
+  PLATFORM: linux/amd64
+
+jobs:
+  build:
+    name: Build image
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        file: [Dockerfile, Dockerfile.slim]
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 1
+
+      - name: Set up Docker Buildx
+        id: buildx
+        uses: crazy-max/ghaction-docker-buildx@v1
+        with:
+          version: v0.3.1
```

Comment:
How did you determine this version? On Marketplace I see this v1.0.4:

![image](https://user-images.githubusercontent.com/6311784/71248809-182ed400-231c-11ea-8b96-cb285e30c748.png)

Reply:
this specifies the docker buildx version, which is currently [v0.3.1](https://github.com/docker/buildx/releases/latest)

Reply:
https://github.com/crazy-max/ghaction-docker-buildx#inputs

Reply:
My mistake. I see the `v1` action tag now